ingest_revisions_dataflow/ingest_utils/run_ingester.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import xml.sax
+import subprocess
+import argparse
+from os import path
+from ingest_utils import wikipedia_revisions_ingester as wiki_ingester
+
+parser = argparse.ArgumentParser(description='Download and save wikipedia revisions from a dump.')
+parser.add_argument('-i', '--input',  help='Path to a wikipedia xml or 7z revision dump file.')
+
+args = parser.parse_args()
+
+
+def run(input_file = None):
+	
+	data_reset_path, revision_reset_path, data_paths = wiki_ingester.get_paths()
```

Comment:
Why bother exposing these 'internal' looking paths when they are only used again in the  wiki_ingester module? Maybe better to expose just a "parse_stream" function that is given the stream, and does the combination of creating paths and parsing? That or the "ParserContentHandler" gets pulled out into a separate library... ?

Reply:
Good point. Done.